lib/manageiq/schema/schema_dumper.rb

```diff
@@ -28,18 +28,18 @@ def add_id_column_comment(table, stream)
       end
 
       # Must be done after all of the table definitions since `metrics_01` is
-      # dumpped prior to `metrics_base`, etc.
+      # dumped prior to `metrics_base`, etc.
       #
-      def miq_metric_table_contraints(stream)
+      def miq_metric_table_constraints(stream)
         inherited_metrics_tables.each do |(table, inherit_from)|
-          match             = table.match(METRIC_ROLLUP_TABLE_REGEXP)
+          child_table_num   = table.match(METRIC_ROLLUP_TABLE_REGEXP)[:CHILD_TABLE_NUM].to_i
```

Comment:
Okay, for anyone else looking over this and at a first glance are confused at what this change was, the only difference here is the addition of the `.to_i`.  The `[:CHILD_TABLE_NUM]` portion was previously below, and the variable name is just being refactored at bit.

I think the reason I split those up is I might have had this as a single method while developing this and the `.match` was also an existence check at the same time (to avoid a `nil` `NoMethodError`), which I think is still being done else where in the dumper.  I think the refactoring is fine, so I have no problem with the change.